docs/DEVELOPMENT.md

```diff
@@ -22,6 +22,23 @@ ENVIRONMENT=local
 
 - Run `pnpm i` at the root to install all the dependencies and run the hooks
 - Run `pnpm local:setup` to run Docker compose dependencies, create databases and migrate database
+
+Solving permission problems on this step:
+
+```bash
+export UID=$(id -u)
+export GID=$(id -g)
+```
+
+Add "user" field to docker-compose.dev.yml
+
+```
+  clickhouse:
+    user: '${UID}:${GID}'
+  db:
+    user: '${UID}:${GID}'
+```
+
```

Comment:
@n1ru4l , not sure if there are also people encountered this kind of problem, can remove, if not needed

Reply:
fine with me though nobody in our team had these issues yet.

Reply:
btw do you docker for container solution?
we use colima, probably, it's because of that